com/win32com/client/build.py

```diff
@@ -363,9 +363,9 @@ def MakeDispatchFuncMethod(self, entry, name, bMakeClass=1):
             defUnnamedArg = "defaultUnnamedArg"
         else:
             linePrefix = ""
-            defNamedOptArg = "pythoncom.Missing"
-            defNamedNotOptArg = "pythoncom.Missing"
```

Comment:
Is just `defNamedNotOptArg` the problem? It would make sense that a non-optional arg might be "empty" rather than "missing".